catalog/ui/catalog-ui-search/src/main/webapp/component/metacard-archive/metacard-archive.hbs

```diff
@@ -11,12 +11,38 @@
  *
  **/
  --}}
-<button class="is-negative archive" data-help="This will remove the item(s) from standard search results.
-To restore archived items, you can click on 'File' in the toolbar,
-and then click 'Restore Archived Items'.">
-    <span class="mainText">Archive item(s)</span>
-    <span class="subText">WARNING: This will remove the item(s) from standard search results.</span>
-</button>
-<button class="is-positive restore" data-help="This will restore the item(s) to standard search results.">
-    <span class="mainText">Restore item(s)</span>
-</button>
\ No newline at end of file
+<div class="is-not-archived">
+    {{#if this.orphans}}
+        <div class="section">
+            <div class="section-title">
+                The following were found to have associations to results in the current selection.  If you continue with archiving, these associations and possibly 
+            more will be orphaned:
+            </div>
+            <div class="section-content">
+                {{#each this.orphans}}
+                    <div>
+                        <a href="#metacards/{{this}}" target="_blank">{{this}}</a>
```

Comment:
✏️ Can we work the titles in here instead of (or in addition to) the IDs? Would be a lot more user-friendly.